lib/src/voip/group_call_session.dart

```diff
@@ -246,16 +246,27 @@ class GroupCallSession {
 
     if (anyJoined.isNotEmpty || anyLeft.isNotEmpty) {
```

Comment:
Is this not redundant?

Reply:
need that for the `onGroupCallEvent.add(GroupCallStateChange.participantsChanged);` below

Reply:
Ah, no, it is not, because the `onGroupCallEvent.add(GroupCallStateChange.participantsChanged);` is still inside of this, but maybe it shouldn't be triggered, if the change was only local...